src/kOS.Safe/Compilation/CompilerOptions.cs

```diff
@@ -4,6 +4,15 @@ namespace kOS.Safe.Compilation
     public class CompilerOptions
     {
         public bool LoadProgramsInSameAddressSpace { get; set; }
+        /// <summary>
+        /// IsCalledFromRun is true when this compile is being made from
+        /// a context in which it was called from a RUN command such that there
+        /// is a chance of there being arguments passed as parameters.  If it is
+        /// the interpreter context, then this should be false to instruct the
+        /// compiler NOT to attempt to look for an arg bottom marker because there
+        /// won't be one.
```

Comment:
The compiler is used to compile interpreter statements one statement at a time.  In that context, there is no arg bottom marker because the interpreter wasn't called with a RUN command.